LDMP/data_io.py

```diff
@@ -175,14 +176,19 @@ def __init__(
         self.out_res = out_res
         self.out_data_type = out_data_type
 
+        # TODO: Confirm if this should be optional and if we should have
```

Comment:
Yes, we should default to WGS84. Using WGS84 would be consistent with past behavior of the plugin so reasonable to use it as a default.